template/android/app/build.gradle

```diff
@@ -121,6 +121,8 @@ def jscFlavor = 'org.webkit:android-jsc:+'
 def enableHermes = project.ext.react.get("enableHermes", false);
 
 android {
+    ndkVersion "20.1.5948944"
```

Comment:
can this be put in the same place as `rootProject.ext.compileSdkVersion`? That way, all versions are put in 1 place?

Reply:
Sounds good I'll update this in the morning!